lib/cli/commands-schema/service.js

```diff
@@ -12,10 +12,14 @@ commands.set('package', {
   usage: 'Packages a Serverless service',
   hasAwsExtension: true,
   options: {
-    package: {
+    'package': {
       usage: 'Output path for the package',
       shortcut: 'p',
     },
+    'minify-template': {
```

Comment:
This is AWS specific setting, so let's add it in `aws-service` context, but also ensuring we use different copy of `package` command configuration over there

Reply:
Hello, thank you for the feedback.

It seems that all of the commands are defined only once (either in `service`, or `aws-service`, or `no-service`). 

When I try to define the `'package'` command in both `service` and `aws-service`, the one in `aws-service` is ignored. Tests still pass, though.

Copied to `aws-service.js`:
```js
commands.set('package', {
  usage: 'Packages a Serverless service',
  options: {
    'package': {
      usage: 'Output path for the package',
      shortcut: 'p',
    },
    'minify-template': {
      usage: 'Minify the CloudFormation template for AWS packages',
      type: 'boolean',
    },
  },
  lifecycleEvents: [
    'cleanup',
    'initialize',
    'setupProviderConfiguration',
    'createDeploymentArtifacts',
    'compileLayers',
    'compileFunctions',
    'compileEvents',
    'finalize',
  ],
});
```

Terminal:
```@Kycermann ➜ .../serverless/dist/mktest/aws-node-project (feat-minify-cf-flag) $ ../../../bin/serverless.js package --help --debug=
process
process: resolve CLI input (no service schema)
process: resolve CLI input (service schema)
process: resolve eventual service configuration
process: service configuration found at /workspaces/serverless/dist/mktest/aws-node-project/serverless.yml
process: service configuration file successfully parsed
process: resolve variables meta
process: resolve CLI input (AWS service schema)
process: construct Serverless instance
process: initialize Serverless instance
process: render help
package                         Packages a Serverless service
--package / -p                  Output path for the package
--region / -r                   Region of the service
--aws-profile                   AWS profile to use with the command
--app                           Dashboard app
--org                           Dashboard org
--use-local-credentials         Rely on locally resolved AWS credentials instead of loading them from Dashboard provider settings (applies only to services integrated with Dashboard)
--config / -c                   Path to serverless config file
--stage / -s                    Stage of the service
--param                         Pass custom parameter values for "param" variable source (usage: --param="key=value")
--help / -h                     Show this message
--version / -v                  Show version info
--verbose                       Show verbose logs
--debug                         Namespace of debug logs to expose (use "*" to display all)

process: finalize { error: undefined, shouldBeSync: undefined, telemetryData: { outcome: 'success' }, shouldSendTelemetry: false }
@Kycermann ➜ .../serverless/dist/mktest/aws-node-project (feat-minify-cf-flag) $
```

It only works when I delete the `'package'` command in `service.js`.

In `aws-service.js` line 300, we have `for (const [name, schema] of serviceCommands) commands.set(name, schema)` which overwrites all commands with commands from `service.js`- so those always take precedence. Not sure how to fix this.

Reply:
Ah I see. This works but `minify-template` is shown at the end of the options (after the generic ones):
```
@Kycermann ➜ /workspaces/serverless/dist (feat-minify-cf-flag) $ ../bin/serverless.js package --help
package                         Packages a Serverless service
--package / -p                  Output path for the package
--region / -r                   Region of the service
--aws-profile                   AWS profile to use with the command
--app                           Dashboard app
--org                           Dashboard org
--use-local-credentials         Rely on locally resolved AWS credentials instead of loading them from Dashboard provider settings (applies only to services integrated with Dashboard)
--config / -c                   Path to serverless config file
--stage / -s                    Stage of the service
--param                         Pass custom parameter values for "param" variable source (usage: --param="key=value")
--help / -h                     Show this message
--version / -v                  Show version info
--verbose                       Show verbose logs
--debug                         Namespace of debug logs to expose (use "*" to display all)
--minify-template               Minify the CloudFormation template for AWS packages```